tests/Util/yaml_fixtures/array_double_quoted_value.test

```diff
@@ -0,0 +1,14 @@
+caddy:
+    ports:
+      - "80:80"
+    volumes:
+      - caddy_data
+===
+$data['caddy']['ports'][] = "443:443";
```

Comment:
Again for purposes of figuring out the root cause, if you changed this from `"443:443"` to `"foo_bar"`, does the error happen?